data/src/test/java/com/linkedin/data/schema/grammar/TestPdlSchemaParser.java

```diff
@@ -237,6 +236,32 @@ public void testNamedDataSchemaWithAliases()
 
   }
 
+  @Test
+  public void testComplexTypeWithProperties()
```

Comment:
Might be overkill, but I'd test to make sure the fields themselves have no properties.

Reply:
done